tests/installation/common_criteria_configuration/cc_disk_encryption_password.pm

```diff
@@ -0,0 +1,19 @@
+# Copyright SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: The test module configure the disk encryption password
+# when using the Common Criteria role.
+# Maintainer: QE Security <none@suse.de>
+
+use parent 'y2_installbase';
+use strict;
+use warnings;
+
+sub run {
+    my $partitioning_scheme = $testapi::distri->get_common_criteria_configuration();
```

Comment:
the name of the variable should correspond with the return of the function.

Reply:
done